From 803bde12d32d16eefeae03422a0ac682e0601c43 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 20 Mar 2017 12:13:30 -0700
Subject: [PATCH 2/2] Define WAIT_ANY if not provided by system

POSIX does not define it and uses -1 directly
some libc do not have this definitions

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 tests/test-ovn.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/tests/test-ovn.c b/tests/test-ovn.c
index 2e82a6f0d..76e7db9fb 100644
--- a/tests/test-ovn.c
+++ b/tests/test-ovn.c
@@ -37,6 +37,10 @@
 #include "simap.h"
 #include "util.h"
 
+#ifndef WAIT_ANY
+# define WAIT_ANY (-1) /* Any process.  */
+#endif
+
 /* --relops: Bitmap of the relational operators to test, in exhaustive test. */
 static unsigned int test_relops;
 
-- 
2.12.0

